theme/less/gouvfr-post.less

```diff
@@ -1,91 +1,6 @@
 body.post-display {
-    .post-container {
-        padding-bottom: 20px;
 
-        .page-header {
-            margin-top: 0px;
-            border: none;
-            margin-bottom: 0px;
-
-            h1 {
-                font-family: @title-font;
-                margin-bottom: 5px;
-                margin-top: 0px;
-
-                @media (max-width: @screen-sm-max) {
-                    font-size: 2em;
-                }
-            }
-
-            small {
-                a {
-                    font-weight: bold;
-                    color: white;
-                }
-
-                a.btn {
-                    vertical-align: text-top;
-                    font-weight: bold;
-                }
-            }
-
-            aside {
-                a:hover .thumbnail, a:active .thumbnail {
-                    text-decoration: none;
-                }
-            }
-        }
-
-        aside {
-            margin-top: 20px;
-
-            > * {
-                margin-bottom: 10px;
-            }
-
-            a.btn {
-                margin-bottom: 5px;
-            }
-        }
-    }
-
-    // correcting main column width
-    @media (min-width: @screen-sm-min) {
-        .smaller {
-            padding-right: @grid-gutter-width * 2.5;
-        }
-    }
-
-    //Action toolbar
-    .main-toolbar {
-        margin-bottom: 20px
-    }
-
-    .post-section {
-
-        .post-headline {
-            font-weight: bold;
-        }
-
-        .post-image {
-            margin: 10px 0;
-
-            position: relative;
-            display: flex;
-            justify-content: center;
-            flex-direction: column;
-            align-items: center;
-
-            img {
-                background-color: white;
-                border: 1px solid @gray-lighter;
-            }
-        }
-    }
-
-    .history-section {
-        min-height: 200px;
+    .nav-section {
```

Comment:
What is this addition related to? I was expecting to see the entire stylesheet to be gone.

Reply:
This color is specific to the `gouvfr` theme, all others statement defines the layout and moved to the core.